README.md

```diff
@@ -1,32 +1,20 @@
 # OpenFn Adaptors ![Build & Test](https://github.com/openfn/adaptors/actions/workflows/ci.yaml/badge.svg?branch=main) ![Build & Test](https://github.com/openfn/adaptors/actions/workflows/docs.yaml/badge.svg?branch=main)
+ 
+The new home for all @openfn [language adaptors](https://docs.openfn.org/adaptors) - open-source Javascript or Typescript modules that provide helper functions to communicate with a specific external system.
 
-The new home for all @openfn language adaptors.
-
-**N.B.: We hope to migrate our 60+ adaptors here over the coming weeks.**
-
-This repo requires [pnpm](https://pnpm.io/installation) to be installed globally
-on your machine.
+For a fully open source workflow automation platform that leverages these adaptors, see [OpenFn Lighnting](https://github.com/OpenFn/lightning).
 
 ## Getting Started
 
-A few first time repo steps:
-
-Install tool versions with [asdf](https://github.com/asdf-vm/asdf)
-
-```
-asdf install
-```
+_Note: This repo requires [pnpm](https://pnpm.io/installation) and [asdf](https://github.com/asdf-vm/asdf) to be installed globally
+on your machine._
 
-Install pnpm:
-
-```
-npm install -g pnpm
-```
-
-And run the setup command:
+A few first time repo steps:
 
 ```
-pnpm run setup
+asdf install #Install tool versions
+npm install -g pnpm #Install pnpm
+pnpm run setup #Run the setup command
```

Comment:
I have a feeling the setup step is redundant now - but I don't really have time to test this today (and it's harmless to leave)